spec/features/user_profile_spec.rb

```diff
@@ -22,8 +22,14 @@
     expect(page).to have_text "Your account has been updated"
   end
 
-  scenario 'can be signed out' do
+  scenario 'can be signed out of' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.